api/groomerFavorites/favRouter.js

```diff
@@ -0,0 +1,44 @@
+//======================//
+//      IMPORTS         //
+//======================//
+const express = require('express');
+const authRequired = require('../middleware/authRequired');
+const groomerFavModel = require('./favModel.js');
+const router = express.Router();
+
+router.all('/', function (req, res, next) {
+  res.header('Access-Control-Allow-Origin', process.env.CORS_ORIGIN);
+  res.header('Access-Control-Allow-Headers', 'X-Requested-With');
+  next();
+});
+
+//============================//
+//  GET all favorite groomers //
+//============================//
+router.get('/', async (req, res) => {
+  try{
+    const data = await groomerFavModel.getAll();
+    res.status(200).json(data);
+  } catch (err){
+    res.status(500).json({ message: err.message})
+  }
+});
+
+//============================//
+//  DELETE groomer by id //
+//============================//
+//router.delete('/:id', authRequired, async (req, res) => {
```

Comment:
is this code important? if not I would just get rid of the commented out code before it gets merged.

Reply:
if it is important, make sure to put in comments next to it what it is used for.

Reply:
The commented out code is important. I made a comment above the code.